test/python/qpy/test_circuit_load_from_qpy.py

```diff
@@ -209,52 +209,6 @@ def test_no_register(self, opt_level):
 class TestVersionArg(QpyCircuitTestCase):
     """Test explicitly setting a qpy version in dump()."""
 
-    def test_custom_gate_name_overlap_persists_with_minimum_version(self):
```

Comment:
This test was removed because it was explicitly testing the bug fixed in QPY 11 was present if we specifically set the QPY version to 10. Since the bug is still fixed in QPY 13, there isn't anything to test anymore since we can't generate a buggy payload.